src/stories/index.stories.js

```diff
@@ -483,6 +484,18 @@ storiesOf('Components', module)
             `
         };
     })
+    .add('Copyable', () => ({
+        components: { Copyable },
+        template: `
+            <div>
+                <Copyable ref="copyable">I'm a text you can copy.</Copyable>
+                <Copyable text="Surprise!!!" style="margin-top: 7rem">When you click me you get a surprise!</Copyable>
+                <button class="nq-button" style="margin-top: 7rem" @click="$refs.copyable.copy()">
+                    Click me to trigger a copy via code
+                </button>
+            </div>
```

Comment:
I think this demo should include a case with children elements, to test that case.